### PR TITLE
Add User-Agent ds3_java_sdk to request headers

### PR DESCRIPTION
**Pull request by GtJosh**

### CODE REVIEW

File: ds3-sdk/src/main/java/com/spectralogic/ds3client/networking/NetworkClientImpl.java [hunk 1]
```diff
@@ -273,6 +273,7 @@ private void addHeaders(final HttpRequest httpRequest) throws IOException, Signa
             httpRequest.addHeader(HOST, NetUtils.buildHostField(NetworkClientImpl.this.connectionDetails));
             httpRequest.addHeader(DATE, date);
             httpRequest.addHeader(CONTENT_TYPE, this.ds3Request.getContentType());
+            httpRequest.addHeader("User-Agent", "ds3_java_sdk");
```
Comment (rpmoore) [inline ds3-sdk/src/main/java/com/spectralogic/ds3client/networking/NetworkClientImpl.java]: This should be a constant like the other headers

Comment (GtJosh) [inline ds3-sdk/src/main/java/com/spectralogic/ds3client/networking/NetworkClientImpl.java]: Got it.

**Comment by rpmoore** [comment]
Have you verified the field is being passed in either with the black pearl logs or through a proxy client?

**Comment by GtJosh** [comment]
Yes, I can see it in fiddler, and double checked the server-wire log.

**Pull request merged**